CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 Change Log
 ===
+v2.3.0
+---
+* [bugs] #401 #367 #324 #352
+* [new-rule] `no-var-keyword` disallows `var` in favor of `let` and `const`
+* [new-rule] `sort-object-literal-keys` forces object-literal keys to be sorted alphabetically
+* Update the `whitespace` rule to work with ES6 module syntax
+* Update `variable-name`, `no-use-before-declare`, and `no-unused-variable` to support destructuring
```

Comment:
let's combine the above 2 bullet points:

* Add support for ES6 destructuring and module syntax (affects `whitespace`, `variable-name`, `no-unused-variable`, `no-use-before-declare` rules)